Chas Advance ESP32/src/wifiHandler.cpp

```diff
@@ -222,7 +222,7 @@ void processBatches(void *parameter)
         Serial.printf("Received batch with %d entries\n", sensorBatch.size());
         for (const auto &entry : sensorBatch)
         {
-          logSensorData(formatUnixTime(entry.timestamp), entry.temperature, entry.humidity, static_cast<ErrorType>(entry.errorType));
+          logSensorData(getTimeStamp(), entry.temperature, entry.humidity, static_cast<bool>(entry.errorType));
```

Comment:
Jag behöver ha följande (ErrorType och inte bool i static_cast) för att koden ska kompilera
```suggestion
          logSensorData(getTimeStamp(), entry.temperature, entry.humidity, static_cast<ErrorType>(entry.errorType));
```